controllers/root.js

```diff
@@ -39,16 +39,23 @@ var Root = (function() {
 
```

Comment:
Magic key (in line 29) is used to allow special clients (for example admins) to pass through the proxy without checking auth. In case the PEP is configured in "validate" mode and the magic key is valid, you have to validate the request.